changelogs/fragments/add_deploy_flask_app_role.yaml

```diff
@@ -0,0 +1,5 @@
+---
+breaking_changes:
```

Comment:
Can we list all the variables that have been changed with old name and the new one? I guess it will be valuable for the release changelog.

Reply:
There are around 20 variables. They are part of the README. I can provide a link to the README here. What do you think?